lisp/ess-dde.el

```diff
@@ -164,7 +164,7 @@ file into an emacs buffer and displays it."
     (sleep-for sleep)
     (if (not bufname)
         (find-file filename)
-      (switch-to-buffer bufname))
+      (pop-to-buffer-same-window bufname))
```

Comment:
What is the semantic difference between these? I cannot figure it out from the docs.

Reply:
The docs are horrible. As far as I can tell, `pop-to-buffer-same-window` is nearly the same as `switch-to-buffer`, but relies on `display-buffer` framework, which `switch-to-buffer` seems to ignore.